docker/nginx/nginx-key.conf

```diff
@@ -51,7 +51,7 @@ http {
         server_name localhost;
 
         # Proxy pass to users service
-        location ~ ^/(users|groups|password|authorize) {
+        location ~ ^/(users|groups|password|authorize|google-callback) {
```

Comment:
Can we have some thing like this 
`auth/callback/{provider}`
Where provider can be google, github, microsoft, etc. 
For future use